packages/changelog-bot/src/main/start.ts

```diff
@@ -31,10 +32,12 @@ export async function start(parameters: {[index: string]: string}): Promise<Chan
     password,
   };
 
-  const changelog = await ChangelogBot.generateChangelog(String(TRAVIS_REPO_SLUG), String(TRAVIS_COMMIT_RANGE));
+  const repoSlug = String(TRAVIS_REPO_SLUG);
```

Comment:
Do you really need to wrap it in an object? There are very few (if any) scenarios where it's needed